thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-comparison-range2/component.js

```diff
@@ -1,7 +1,12 @@
 import { computed } from '@ember/object';
```

Comment:
This file name looks fishy "rootcause-select-comparison-range2"

Reply:
ask @aaronucsd :-)

Reply:
It is a POC route. People wanted to try it live (designer etc). We can rename to have `poc` in the name